### ISSUE TITLE
Anti-Sanctioned-Spam

### ISSUE THREAD

**Issue by NanoCats** [night/betterttv#4474]
I'm not referring to "Actual" spam such as somebody posting a link to a shady scam site but, I'd like a function to automatically block "Sanctioned" spam, by that I mean messages like automated meaningless ones posted by bots that contain 50 rainbow animated emotes IE 10+ lines of 

"HYPE :SomeGuyHype: HYPE :SomeGuyHype: HYPE :SomeGuyHype: HYPE :SomeGuyHype: HYPE :SomeGuyHype: HYPE :SomeGuyHype: HYPE :SomeGuyHype: "

Basically it would just have some simple criteria you could set to reduce the messages, Something like this but [] is a number you can set
![image](https://user-images.githubusercontent.com/14065903/115975193-ae869180-a530-11eb-91d1-ad1217799d90.png)

**Comment by night** [comment]
duplicate of #1922
